client/src/app/+videos/+video-edit/video-add-components/video-import-torrent.component.html

```diff
@@ -25,19 +25,30 @@
 
     <div class="form-group">
       <label i18n for="first-step-channel">Channel</label>
-      <div class="peertube-select-container">
-        <select id="first-step-channel" [(ngModel)]="firstStepChannelId" class="form-control">
-          <option *ngFor="let channel of userVideoChannels" [value]="channel.id">{{ channel.label }}</option>
-        </select>
+      <div class="peertube-ng-select-container">
+        <ng-select
```

Comment:
Could you create a dedicated component to choose user channels?

Reply:
@Chocobozzz should I put that in `client/src/app/shared/shared-forms/select-{channel,options}`?

Reply:
Yep